README.md

```
@@ -1,96 +1,91 @@
 
-For Hardware:
-- [List main components]
-- [List specifications]
-- [List tools required]
 
 ### Implementation
 For Software:
 # Installation
-[commands]
+
+git clone https://github.com/your-username/vazhi-illae-kuzhi.git
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)